extensions/custom-field/shopify.extension.toml

```diff
@@ -1,7 +1,7 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
-api_version = "2023-10"
+api_version = "unstable"
```

Comment:
Is that ok to use `"unstable"` in tutorials? The `useDeliveryGroupListTarget` hook won't work until `"2024-07"`, which isn't available yet, so the example will only be usable with `"unstable"` for now.